packages/layout/src/Box/Box.tsx

```diff
@@ -1,64 +1,35 @@
-import { useComposedRefs } from "@telegraph/compose-refs";
 import type {
   PolymorphicPropsWithTgphRef,
   TgphElement,
 } from "@telegraph/helpers";
+import { useStyleProps } from "@telegraph/style-engine";
 import clsx from "clsx";
-import React from "react";
 
-import { propsToCssVariables } from "../helpers/css-variables";
-
-import { BOX_PROPS } from "./Box.constants";
-import { BoxPropsTokens } from "./Box.types";
+import { StyleProps, stylePropsFn } from "./Box.css";
 
 type BoxProps<T extends TgphElement> = PolymorphicPropsWithTgphRef<
   T,
   HTMLElement
 > &
-  BoxPropsTokens;
+  StyleProps;
 
 const Box = <T extends TgphElement>({
   as,
   className,
   tgphRef,
+  borderColor = "gray-4",
```

Comment:
Is this right? Should have this as the default for all boxes?

Reply:
Yeah I think this is a good default so that you don't have to apply the color yourself, instead we just use the color we use as border colors in 99% of places so you can just define the widtha.

Reply:
ohhh got it. yeah agree.